day_1_ddo/ddo_hackathon/admin/app.py

```diff
@@ -9,22 +9,24 @@
 from google.oauth2 import service_account
 
 
-def save_valid_submission(code, team_name, bucket_name="ddo_hackathon"):
+def save_valid_submission(code, team_name, track, bucket_name="ddo_hackathon"):
```

Comment:
I think these changes were already made in the other PR. I suggest we wait until we merged the other two and then rebase. I will review this PR then if that's ok @EmPajak21 :)